src/Symfony/Component/Security/Http/RememberMe/AbstractRememberMeServices.php

```diff
@@ -124,7 +125,19 @@ public function getSecret()
                 $this->logger->info('Remember-me cookie accepted.');
             }
 
-            return new RememberMeToken($user, $this->providerKey, $this->secret);
+            $token = new RememberMeToken($user, $this->providerKey, $this->secret);
+
+            if (!$token->isAuthenticated()) {
```

Comment:
Won't the token always be not authenticated because `setAuthenticated()` is now never called?

Reply:
yeah, just found that out.... working on a fix